src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -120,10 +164,23 @@
             }
 
             return {
-                tabs: tabs
+                tabs: tabs,
+                areaFilterId: "area"
             }
         },
         computed: {
+            filterChoroplethIndicators(){
```

Comment:
Had a little bit of trouble working out what these were based on the name - is "filter" here being used as a verb? Although computeds are defined as functions they're really thought of as objects, so `filteredChoroplethIndicators` etc would be the convention (if I've correctly understood what these are!) 

And is there a type that can be used in the lambda rather than `any`?